README.md

```diff
@@ -194,7 +194,7 @@ maba_webpack:
             type:                 twig_directory # analyses twig templates inside given directory
             resource:             %kernel.root_dir%/Resources/views
     twig:
-        function_name:        webpack_asset     # function name in twig templates
+        function_prefix:      webpack_webpack   # function prefix in twig templates
```

Comment:
Not sure it's good idea but I think it's better than configuring name of every single function.